vcx/libvcx/src/utils/libindy/wallet.rs

```diff
@@ -14,15 +14,14 @@ pub static mut WALLET_HANDLE: i32 = 0;
 
 pub fn get_wallet_handle() -> i32 { unsafe { WALLET_HANDLE } }
 
-pub fn create_wallet(wallet_name: &str, wallet_type: Option<&str>) -> Result<(), u32> {
+pub fn create_wallet(wallet_name: &str, wallet_type: Option<&str>, storage_config: Option<&str>, storage_creds: Option<&str>) -> Result<(), u32> {
```

Comment:
There's an instance of this in the tests for src/api/vcx.rs that hasn't been updated. (also other instances of delete_wallet and init_wallet).

Change Cargo.toml to have:

default = ["agency","pool_tests"]

To see the problems.

Reply:
@glowkey   thanks for the review, files are updates.